src/Symfony/Component/Notifier/Bridge/Bluesky/composer.json

```
@@ -0,0 +1,36 @@
+{
+    "name": "symfony/bluesky-notifier",
+    "type": "symfony-notifier-bridge",
+    "description": "Symfony Bluesky Notifier Bridge",
+    "keywords": ["bluesky", "bluesky", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        },
+        {
+            "name": "Tobias Nyholm",
+            "email": "tobias.nyholm@gmail.com"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "psr/log": "^1|^2|^3",
+        "symfony/http-client": "^6.4|^7.0",
+        "symfony/notifier": "^6.4|^7.0",
```

```suggestion
        "symfony/notifier": "^7.1",
```

Please adjust `UnsupportedSchemeException` and ÙnsupportedSchemeExceptionTest`, that's why we need to raise the bar to 7.1